client/app/styles/less/sf-additional.less

```diff
@@ -264,8 +264,8 @@
     	margin-top: 0;
 	}
 	.separator-handler{
-	    margin-top: 0;
-    	margin-right: 20px;
+		margin-top: 0;
+		margin-right: 10px;
```

Comment:
I recommend setting your editor to insert 4 spaces instead of a `<tab>` character - it quickly becomes messy if tabs and spaces are mixed in the same file.